README.md

```diff
@@ -1,20 +1,46 @@
-# Catone, the OpenCore config.plist censor
+![macOS](https://img.shields.io/badge/Supported_OC_build:-≥0.8.2-white.svg)
 
-The following script was created to censor sensitive fields in OpenCore's config.plist
+# OC Anonymizer – remove sensitive Data from your config.plist
 
-The reason why I created it is that many EFI I find online contain sensitive fields such as SMBIOS data.
-With this script you won't have any prob at all
+## About
+Python Script for removing sensitive Data from OpenCore's `config.plist` before sharing your Config/EFI folder online. It also changes some settings to default, such as: ScanPolicy, Custom Entries, LauncherOptions, etc. See details below. Basically it removes all those settings specific to your system and user preferences which may not be desireble to have on a different machine.
 
-# Usage
+## Features
 
-```bash
-$> python3 oc_anonymizer.py PATH_TO_CONFIG.plist
-```
+Changes the following Settings/Parameters in the **config.plist**:
 
-It'll create `censored_config.plist` with the stripped sensitive data
+- **SMBIOS** data:
```

Comment:
I wouldn't say `deletes` but rather `censor` as it actually replace each value with a non-valid one